<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_score</class>
 <widget class="QDialog" name="edit_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑成绩</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>231</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>231</width>
     <height>32</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>421</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>语文：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>数学：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>英语：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>同学互评:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>思想品德:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>教师评价</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="id_edit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="chinese_edit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="math_edit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="english_edit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="mutual_edit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="quality_edit"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="teacher_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*学号必须存在</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*成绩在0-100之间</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
